src/components/PivotTable/PivotTableValueCell.js

```diff
@@ -33,7 +27,7 @@ export const PivotTableValueCell = ({
         return (
             <PivotTableEmptyCell
                 type={cellContent?.cellType}
-                onClick={onClick}
+                onClick={onToggleContextualMenu ? onClick : undefined}
```

Comment:
@edoardo I think `onClick` isn’t defined yet here so this will break on empty cells?

Reply:
True, fixing in a new PR.